src/App.js

```diff
@@ -7,26 +7,26 @@ import TeamPage from "./Pages/TeamPage";
 import MatchupPage from "./Pages/MatchupPage";
 import FreeAgentsPage from "./Pages/FreeAgentsPage";
 import LeaguePage from "./Pages/LeaguePage";
+import H2P from './Pages/H2P';
+import NBALiveStats from './Pages/NBALiveStats';
+import LandingPage from './Pages/LandingPage';
+import NHLLiveStats from './Pages/NHLLiveStats';
```

Comment:
Why is this in the pages folder. Clean it up bud

Reply:
still pages